recipes/f2c/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "f2c" %}
+{% set version = "20160102" %}
+{% set sha256 = "fc287517edc67c184bf91860aaa7b342c6c8ce008b298c2a08c4616133009c21" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar
+  url: http://netlib.sandia.gov/cgi-bin/netlib/netlibfiles.tar?filename=netlib/{{name}}
```

Comment:
Is this the official source?

Reply:
Good point; think it should be http://www.netlib.org/f2c/src.tgz.

Reply:
Yes, 1 is the easiest

Reply:
yes, that's exactly the reason why I am using the mirror. It is the only place where you can get the whole thing at once.

The only other option I see is to split into two packages (`f2c` and `libf2c`) each one with its own source, but that doesn't seem ideal to me given that you cannot use a program compiled with `f2c` without `libf2c`.